src/components/app/updateUserProfileForm/step1/constants.ts

```diff
@@ -0,0 +1,14 @@
+import { ORDERED_SUPPORTED_COUNTRIES } from '@/utils/shared/supportedCountries'
+
+export const DEFAULT_DISCLAIMER =
+  'You are about to change your viewing country based on the address you have chosen'
+
+export const DISCLAIMERS_BY_COUNTRY_CODE: Record<
+  (typeof ORDERED_SUPPORTED_COUNTRIES)[number],
```

Comment:
No need for this, you can just use `SupportedCountryCodes`
```suggestion
  SupportedCountryCodes,
```